### PR TITLE
refactor: move scripts to root and use pnpm

### PR DESCRIPTION
**Pull request by raisedadead**
This PR moves the script packages into the repo root and uses pnpm. In a later PR, we can consolidate some of these, like `chat` and `chat-emails` and, of course, anything else that is broken. 

In theory, nothing should have because they were never interdependent.

### CODE REVIEW

File: tricky-challenges/package.json
Comment (ShaunSHamilton) [inline tricky-challenges/package.json:1]: This workspace needs a peerdep of `@types/node` installed.

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Single borken point I could find. Seems unrelated, but is still braking due to workspaces.

Which brings me to: Do we really want all of these unrelated scripts to be a part of a workspace?

To be honest, I just dislike the need to install 100s of unrelated root npm packages just because I wanted to run a single script with 2 root deps.

**Comment by raisedadead** [comment]
Agreed. I will update this in the morning.

**Comment by ojeytonwilliams** [comment]
Given that it takes all of 3 seconds to install everything, I'd just go with workspaces. It seems simpler to me.

> install 100s of unrelated root npm packages just because I wanted to run a single script with 2 root deps.

Even that shouldn't be a problem if https://github.com/pnpm/pnpm/issues/6300 is fixed.

**Review by ShaunSHamilton — APPROVED** [review]
I am happy with this 👍

**Comment by raisedadead** [comment]
Merging because I want to add a few scripts I have lying around.

**Pull request merged**